src/test/java/com/codurance/retropolis/acceptance/board/BoardStepDefsIntegrationTest.java

```diff
@@ -52,4 +58,46 @@ public void theClientReceivesBoardWithThreeColumnsAnd(String firstTitle, String
             new Column(3L, thirdTitle, Collections.emptyList()))))));
   }
 
+  @Given("a user has accessed the test board")
+  public void aUserHasAccessedTheTestBoard() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, TOKEN);
+    HttpWrapper.executeGet(url + "/boards/1", headers);
+  }
+
+  @And("a user has previously accessed the board")
+  public void aUserHasPreviouslyAccessedTheBoard() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, TOKEN);
+    HttpWrapper.executeGet(url + "/boards/1", headers);
+  }
+
+  @When("the user requests all their boards")
+  public void theUserRequestsAllTheirBoards() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, TOKEN);
+    HttpWrapper.executeGet(url + "/boards", headers);
+  }
+
+  @Then("the user receives a list of the boards with one called {string}")
+  public void theUserReceivesAListOfTheBoardsWithOneCalled(String title)
+      throws JsonProcessingException {
+    assertThat(HttpWrapper.responseResult.getResponseCode(), is(200));
+
+    List<Board> boards = new ObjectMapper().readValue(responseResult.getBody(), new TypeReference<>() {
+    });
+
+    assertThat(boards.size(), is(1));
+    assertThat(boards.get(0).getTitle(), is("test board"));
```

Comment:
You could use the parameter you are passing to the function instead of `test board`